man/ukify.xml

```diff
@@ -373,6 +375,30 @@
           <xi:include href="version-info.xml" xpointer="v253"/></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>DeviceTreeAuto=<replaceable>PATH</replaceable>...</varname></term>
+          <term><option>--devicetree-auto=<replaceable>PATH</replaceable></option></term>
+
+          <listitem><para>Zero or more automatically selectable DeviceTree files. In the configuration file, items are separated by
+          whitespace. Each DeviceTree will be in a separate <literal>.dtbauto</literal> section.</para>
+
+          <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><varname>HWIDs=<replaceable>PATH</replaceable></varname></term>
+          <term><option>--hwids=<replaceable>PATH</replaceable></option></term>
+
+          <listitem><para>The hardware ID device table (the <literal>.hwids</literal> section). The argument is a
+          path to a directory with JSON HWID device description files. Each JSON file needs to contain <literal>name</literal>
+          and <literal>compatible</literal> strings and also array of CHIDs/HWIDs as <literal>hwids</literal>.
+          If not specified, the section will not be present. It is recommended to specify this parameter if automatically
+          selectable DeviceTrees are to be used.
```

Comment:
This could use a link to some spec, because reading this, I have no idea what a `compatible` string is, i.e. what it must or may contain; same for the name.

Reply:
```suggestion
          path to a directory with JSON HWID device description files. Each file needs to contain single JSON object with a <literal>name</literal>, <literal>compatible</literal> and <literal>hwids</literal> keys. The <literal>name</literal> and <literal>compatible</literal> keys must have string values and the <literal>hwids</literal</literal> key must have a list of strings as value, where the strings must be valid UUIDs that represent CHIDs/HWIDs.
          If not specified, the section will not be present. It is recommended to specify this parameter if automatically
          selectable DeviceTrees are to be used.
```